ais-infra-alerting/template.yaml

```diff
@@ -40,13 +40,22 @@ Parameters:
     ConstraintDescription: "must be an AWS Chatbot Slack workspace ID"
   SlackChannelId:
     Description: >
-      The ID of the Slack channel where alert notification messages are posted.
+      The ID of the Slack channel where alert notification messages are posted from our non-prod environments.
       This is taken from the channel details in Slack.
       Default is the di-accounts-bravo-ais-notifications channel.
     Type: String
     Default: "C065MT3RKV4" # di-accounts-bravo-ais-notifications
     AllowedPattern: "\\w+"
     ConstraintDescription: "must be a Slack channel ID"
+  SlackChannelIdProd:
```

Comment:
Given this is only referenced in `SlackChannelId:` you could just provide the slack channel IDs there rather than having two separate parameters

Reply:
Yeah can do if that also work and add a comment to highlight which slack channel it's referring to there 👍